src/scss/layout/_title.scss

```diff
@@ -52,3 +52,16 @@
     }
   }
 }
+
+span.first {
+  display: block;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: var(--h3);
+  font-weight: 300;
```

Comment:
I don't think the mockup calls for a special font-weight? I think this could be `normal`.